lib/rails_live_reload/engine.rb

```diff
@@ -1,7 +1,11 @@
 module RailsLiveReload
   class Railtie < ::Rails::Engine
-    if RailsLiveReload.enabled? && (defined?(::Rails::Server) || ENV['SERVER_PROCESS'])
-      initializer "rails_live_reload.middleware" do |app|
+    def enabled?
```

Comment:
`enabled` can probably be changed into something more meaningful. Open to suggestions.